models/base_model.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python3
```

Comment:
You should not push this file as Goodness James had already pushed a file with same name. This will lead to merge conflicts.